### PR TITLE
allow to configure NAT

### PR DESCRIPTION
**Pull request by fblackburn1**
why: our new CI spawn server behind a NAT

### CODE REVIEW

File: wazo_acceptance/prerequisite.py [hunk 1]
```diff
@@ -266,6 +269,25 @@ def _configure_consul(context):
         context.remote_sysutils.restart_service('consul')
 
 
+def _configure_nat(context):
+    if not context.wazo_config['prerequisites']['nat']['enabled']:
+        logger.debug('Skipping NAT configuration')
```
Comment (sduthil) [inline wazo_acceptance/prerequisite.py:274]: Missing a `return`?

File: wazo_acceptance/prerequisite.py [hunk 2]
```diff
@@ -266,6 +269,25 @@ def _configure_consul(context):
...
+
+    transport_udp = context.helpers.sip_transport.get_by(name='transport-udp')
+    external_ip = context.wazo_config['wazo_host']
```
Comment (sduthil) [inline wazo_acceptance/prerequisite.py:277]: As discussed, we need an IP address here, but `wazo_host` may be a hostname, not an IP address.

File: wazo_acceptance/helpers/sip_transport.py [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+        for transport in transports:
+            return transport
+
+    def update(self, body):
+        with self._context.helpers.bus.wait_for_asterisk_reload(pjsip=True):
+            self._confd_client.sip_transports.update(body)
+
+    def update_options(self, sip_transport, new_option, value):
+        options = sip_transport['options']
+        new_options = [option for option in options if option[0] != new_option]
```
Comment (sduthil) [inline wazo_acceptance/helpers/sip_transport.py:28]:
```suggestion
        new_options = [(name, value) for option in options if name != new_option]
```

**Review by sduthil — CHANGES_REQUESTED** [review]

File: wazo_acceptance/prerequisite.py (hunk 1 above)
Comment (fblackburn1) [reply wazo_acceptance/prerequisite.py:274]: oh

File: wazo_acceptance/prerequisite.py (hunk 2 above)
Comment (fblackburn1) [reply wazo_acceptance/prerequisite.py]: ye

File: wazo_acceptance/helpers/sip_transport.py (hunk 3 above)
Comment (fblackburn1) [reply wazo_acceptance/helpers/sip_transport.py]: ah

**Review by sduthil — APPROVED** [review]

**Pull request merged**